tests/foreman/cli/test_host.py

```diff
@@ -2207,7 +2207,7 @@ def setUpClass(cls):
             u'lifecycle-environment-id': cls.env['id'],
             u'activationkey-id': cls.activation_key['id'],
             u'subscription': cls.subscription_name,
-        })
+        }, force_use_cdn=True)
```

Comment:
@abalakh we are not testing the packages them selves, or what can do, only that that repos can be enabled and one of the packages from that installed.